test/irb/test_history.rb

```diff
@@ -138,10 +150,11 @@ def test_history_concurrent_use_not_present
       IRB.conf[:SAVE_HISTORY] = 1
       Dir.mktmpdir("test_irb_history_") do |tmpdir|
         ENV["HOME"] = tmpdir
-        io = TestInputMethodWithHistory.new
+        io = TestInputMethodWithRelineHistory.new
         io.class::HISTORY.clear
         io.load_history
-        io.class::HISTORY.concat(%w"line1 line2")
+        io.class::HISTORY << 'line1'
+        io.class::HISTORY << 'line2'
```

Comment:
`concat` isn't available for `Readline::HISTORY`.  I updated this logic so that this test could use `TestInputMethodWithReadlineHistory` if we wanted.